tests/rptest/services/producer_swarm.py

```diff
@@ -110,7 +120,7 @@ def start_node(self, node, clean=None):
         cmd = f"RUST_LOG={self._log_level} bash /opt/remote/control/start.sh {self.EXE} \"{cmd}\""
         node.account.ssh(cmd)
         self._redpanda.wait_until(
-            lambda: self.is_alive(node),
+            lambda: self.is_alive(),
```

Comment:
The `lambda` does look clear.  I wonder if just giving `self.is_alive` would just work now.

Reply:
It does, changed in https://github.com/redpanda-data/redpanda/compare/5d9ad58e47691dbc16c82a9e1b5b413cab4307e0..837694d18838e6329dab858c39c2e930e831ccfc